forge-gui/res/cardsfolder/upcoming/verdant_dread.txt

```diff
@@ -0,0 +1,7 @@
+Name:Verdant Dread
+ManaCost:1 G
+Types:Enchantment
+T:Mode$ ChangesZone | ValidCard$ Card.Self,Card.namedVerdant Dread | Origin$ Any | Destination$ Battlefield | Execute$ TrigManifestDread | TriggerDescription$ Whenever CARDNAME or another card named CARDNAME enters, manifest dread.
+SVar:TrigManifestDread:DB$ ManifestDread
+A:AB$ MakeCard | Cost$ 3 G G | Conjure$ True | Name$ Verdant Dread | Zone$ Battlefield | SorcerySpeed$ True | SpellDescription$ Conjure a card named CARDNAME onto the battlefield. Activate only as a sorcery.
```

Comment:
the "named X" parts are constants

Reply:
the first instance was correct :p

Reply:
I modified it back to original, is there something else that needs changed?

Reply:
uhh I think you misunderstood again
I'll edit it correctly, let me know if you need more details